helpers/cli.py

```diff
@@ -14,28 +15,100 @@
 
 
 class CLI(object):
-    NO_COLOR = '\033[0m'
-    COLOR_ERROR = '\033[91m'
-    COLOR_SUCCESS = '\033[92m'
-    COLOR_INFO = '\033[94m'
-    COLOR_WARNING = '\033[95m'
+
+    NO_COLOR = '\033[0;0m'
+    COLOR_ERROR = '\033[0;31m'  # dark red
+    COLOR_SUCCESS = '\033[0;32m'  # dark green
+    COLOR_INFO = '\033[1;34m'  # blue
+    COLOR_WARNING = '\033[1;31m'  # red
+    COLOR_QUESTION = '\033[1;33m'  # dark yellow
+    COLOR_DEFAULT = '\033[1;37m'  # white
 
     EMPTY_CHARACTER = '-'
 
+    DEFAULT_CHOICES = {
+        '1': True,
+        '2': False,
+    }
+    DEFAULT_RESPONSES = dict((v, k) for k, v in DEFAULT_CHOICES.items())
```

Comment:
Yeah, Indeed. 
I had this idea in case `DEFAULT_CHOICES` would get more options in the future. 
What about that? 

```python3
    # We need an inverted dict version of `DEFAULT_CHOICES` to be able to
    # retrieve keys from the values
    DEFAULT_RESPONSES = dict(zip(DEFAULT_CHOICES.values(),
                                 DEFAULT_CHOICES.keys()))
```

Moreover `DEFAULT_RESPONSES` is only used in `get_response()` and could be moved there as local variable. The drawback: it would be evaluated each time `get_response()` is called.

What do you think?

Reply:
`CLI.get_response()` is called roughly 48 times at the moment, so it might be unnecessary computation to put in there. If we did though, the `zip` approach is definitely faster. If DEFAULT_CHOICES has 20 options, here's the comparison:

- `dict((v, k) for k, v in DEFAULT_CHOICES.items())`: `4.35 µs ± 76.4 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)`
- `dict(zip(DEFAULT_CHOICES.values(), DEFAULT_CHOICES.keys()))`: `2.38 µs ± 52.8 ns per loop (mean ± std. dev. of 7 runs, 100000 loops each)`

We can maybe go with the `zip`, keeping it where it currently is but having that comment to explain what's happening 👍